node/tests/staged/narwhal.yaml

```diff
@@ -52,10 +52,10 @@ Header:
     - round: U64
     - epoch: U64
     - payload:
-        MAP:
-          KEY:
-            TYPENAME: BatchDigest
-          VALUE: U32
+        SEQ:
+          TUPLE:
+            - TYPENAME: BatchDigest
+            - U32
```

Comment:
Ah so it serialises as a sequence of tuples!

Reply:
Yes looks like it but I agree that we can change it into a vector